[docs] Update concurrencies info in FAQ (#30296)
expo/expo: docs/pages/billing/faq.mdx

### How do I increase build concurrencies on my account?

> If you're already subscribed to a paid EAS plan, you can buy additional concurrencies in [the Add-ons section on the Billing](https://expo.dev/settings/billing#addons) page.

Reviewer: ```suggestion
If you're already subscribed to a paid EAS plan, you can buy additional concurrencies in the [**Add-ons** section](https://expo.dev/settings/billing#addons) under **Billing**.
```

Let's make settings/pages name bold as this is the pattern we follow in this doc for describing paths to other settings/areas of billing page.

> If you're still on the Free plan, you will need to set up a paid subscription. [Click here](https://expo.dev/accounts/[account]/settings/billing/cart) to choose a plan. You can select a number of additional concurrencies to add to your subscription on the checkout page.

Reviewer: ```suggestion
If you're on the Free plan, you'll need to set up a paid subscription. [Click here](https://expo.dev/accounts/[account]/settings/billing/cart) to choose a new plan. Then, select a number of additional concurrencies to add to your subscription on the checkout page.
```

"If you're still" sounds like a bit aggressive to me. I think we remove the word "still" here.
